core/main/src/broker/broker_utils.rs

```diff
@@ -111,4 +112,68 @@ impl BrokerUtils {
             data: None,
         }))
     }
+
+    pub async fn get_language(ctx: &CallContext, state: &PlatformState) -> RpcResult<String> {
```

Comment:
Make this function generic by accepting `method` name as an input param. After that,  change the usage in caller by passing `localization.language` or `localization.countryCode` as param to the new function.  Remove `get_language` and `get_country_code` functions from BrokerUtils